src/praisonai/praisonai/framework_adapters/praisonai_adapter.py

```
@@ -59,6 +59,55 @@ def run(
+                interactive_loop = asyncio.new_event_loop()
+                try:
+                    interactive_loop.run_until_complete(interactive_runtime.start())
+                    
+                    centric_tools = create_agent_centric_tools(interactive_runtime)
+                    logger.info(f"Loaded {len(centric_tools)} InteractiveRuntime tools")
+                    tools_list.extend(centric_tools)
+                    
+                finally:
+                    try:
+                        interactive_loop.run_until_complete(interactive_runtime.stop())
+                    except Exception as stop_error:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)